administrator/language/de-DE/mod_popular.ini

```diff
@@ -8,12 +8,23 @@ MOD_POPULAR="Beiträge – Beliebte"
 MOD_POPULAR_CREATED="Erstellt"
 MOD_POPULAR_FIELD_AUTHORS_LABEL="Autoren"
 MOD_POPULAR_FIELD_COUNT_LABEL="Anzahl"
+MOD_POPULAR_FIELD_DATEFILTERING_LABEL="Datumsfilter"
+MOD_POPULAR_FIELD_DATERANGEFIELD_LABEL="Datumsbereich"
```

Comment:
```suggestion
MOD_POPULAR_FIELD_DATERANGEFIELD_LABEL="Zeitraum"
```

Reply:
Wir verwenden bisher Datumsbereich.
Allerdings ist die Änderung sinnvoll. Sollte aber in einem Clean-Up-PR gemacht werden, für alle Vorkommnisse.